src/index.ts

```diff
@@ -297,15 +298,14 @@ export default class NestedList {
 
     if (this.listStyle === 'ordered') {
       const startWithElement = renderToolboxInput(
-        (index: number) => this.changeStartWith(index),
+        (index: string) => this.changeStartWith(Number(index)),
         {
           value: String(this.data.start ?? 1),
           placeholder: '',
           attributes: {
-            type: 'number',
-            step: '1',
```

Comment:
why they were removed?

Reply:
Because now we use sanitizer, we do not need to check that inside of the input there are only numbers

Also when type is set to number, printing the dot does not trigger on input event listener (input of the dot and comma is treated as temporary input idk seems like this behaviour is related only to chrome)